snap/info_test.go

```diff
@@ -824,15 +824,35 @@ hooks:
 func (s *infoSuite) TestReadInfoFromSnapFileCatchesInvalidImplicitHook(c *C) {
 	yaml := `name: foo
 version: 1.0`
-	snapPath := snaptest.MakeTestSnapWithFiles(c, yaml, emptyHooks("123abc"))
```

Comment:
`MakeTestSnapWithFiles` uses snap pack. With the new hook checking capability this fails to produce a snap, therefore used mocking instead.

Reply:
are other packages outside of snap affected by anything similar? wondering as the PR didn't get to run full unittests

Reply:
All unit tests run, the problem is limited to this instance only.
unit-test are blocked by golang-cli check not covered with run-checks, i think Oliver aligned this, I may need to rebase.